metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -459,6 +460,22 @@ class Compilers(
             }
           }
 
+          /**
+           * ScalaCli scripts begin with
+           * `object script{
+           * /*<script>*/`
```

Comment:
This might change from time to time so the safest is to use AsjsutData and drop anything that has offset <0

Reply:
Currently `AdjustData` didn't remove the `/*<script>*/` comment, so I changed it in `MappedSource`